airflow/utils/dag_processing.py

```diff
@@ -426,7 +426,7 @@ def harvest_simple_dags(self):
         :return: List of parsing result in SimpleDag format.
         """
         # Receive any pending messages before checking if the process has exited.
-        while self._parent_signal_conn.poll():
+        while self._parent_signal_conn.poll(timeout=0.01):
```

Comment:
What is the default timeout? Is it `0`?

Reply:
Yes, which says check but don't wait. (None is wait forever)